packages/core/src/util/normalizeConfig.js

```diff
@@ -67,6 +68,7 @@ const defaults = {
   tracing: {
     enabled: true,
     automaticTracingEnabled: true,
+    activateImmediately: false,
```

Comment:
We probably should consider adding this config option (and the corresponding env var) to our docs.